### ISSUE TITLE
Feature: support new Playwright tags

### ISSUE THREAD

**Issue by vitalets** [vitalets/playwright-bdd#98]
**The problem**
Currently Cucumber tags are not converted to Playwright tags.
We were waiting for Playwright Tags API here https://github.com/microsoft/playwright/issues/23180 and now it's released.
Playwright 1.42 supports tags :)

[New syntax](https://playwright.dev/docs/test-annotations#tag-tests) for adding tags to the tests (@-tokens in the test title are still supported):
```ts
test('test customer login', {
tag: ['https://github.com/fast', '@login'],
}, async ({ page }) => {
// ...
});
```
Use --grep command line option to run only tests with certain tags.
```
npx playwright test --grep https://github.com/fast
```
**A solution**
Add Cucumber tags to generated test files for Playwright >= 1.42.

**Comment by vitalets** [comment]
With https://github.com/microsoft/playwright/issues/29793 we will be able to access Playwright tags inside step definitions, that is the same as current `$tags` fixture.

**Comment by NikkTod** [comment]
Nice, they were quick with https://github.com/microsoft/playwright/issues/29793, its now implemented in beta :)

**Comment by vitalets** [comment]
Released in **v6.1.0**.
Feel free to report in case of any issues.
